src/components/PageSettingsModal/ResourcePageSettingsModal.tsx

```diff
@@ -242,6 +243,7 @@ export const ResourcePageSettingsModal = ({
                 onChange={() => {
                   trigger("permalink")
                   trigger("external")
+                  trigger("title")
```

Comment:
could i get more context on why a code change is required for e2e to pass?

Reply:
Since we dont refresh on focus anymore, the title displayed was stale, so we effectively need to trigger a refetch here